src/commands/lambda/uninstrument.ts

```diff
@@ -2,27 +2,74 @@ import {CloudWatchLogs, Lambda} from 'aws-sdk'
 import {bold, cyan, red, yellow} from 'chalk'
 import {Command} from 'clipanion'
 import {parseConfigFile} from '../../helpers/utils'
-import {collectFunctionsByRegion, updateLambdaFunctionConfigs} from './functions/commons'
+import {AWS_DEFAULT_REGION_ENV_VAR} from './constants'
+import {
+  collectFunctionsByRegion,
+  getAllLambdaFunctionConfigs,
+  isMissingAWSCredentials,
+  updateLambdaFunctionConfigs,
+  willUpdateFunctionConfigs,
+} from './functions/commons'
 import {getUninstrumentedFunctionConfigs, getUninstrumentedFunctionConfigsFromRegEx} from './functions/uninstrument'
 import {FunctionConfiguration} from './interfaces'
+import {requestAWSCredentials, requestChangesConfirmation, requestFunctionSelection} from './prompt'
 
 export class UninstrumentCommand extends Command {
   private config: any = {
     functions: [],
-    region: process.env.AWS_DEFAULT_REGION,
+    region: process.env[AWS_DEFAULT_REGION_ENV_VAR],
   }
   private configPath?: string
   private dryRun = false
   private forwarder?: string
   private functions: string[] = []
+  private interactive = false
   private regExPattern?: string
   private region?: string
 
   public async execute() {
     const lambdaConfig = {lambda: this.config}
     this.config = (await parseConfigFile(lambdaConfig, this.configPath)).lambda
 
-    const hasSpecifiedFuntions = this.functions.length !== 0 || this.config.functions.length !== 0
+    let hasSpecifiedFuntions = this.functions.length !== 0 || this.config.functions.length !== 0
+    // Trial user experience
```

Comment:
Nit: I don't think you need this "trial user experience" comment, the code should speak for itself as much as possible